backend/src/main/java/com/bakdata/conquery/sql/conversion/model/ColumnDateRange.java

```diff
@@ -105,4 +101,21 @@ public ColumnDateRange as(String alias) {
 		);
 	}
 
+	public Condition join(ColumnDateRange right) {
+		if (this.isSingleColumnRange() != right.isSingleColumnRange()) {
+			throw new UnsupportedOperationException("Can only join ColumnDateRanges of same type");
+		}
```

Comment:
in Hana können wir ja eh keine singleColumnRanges machen oder? Können wir dann vlt einen Standard erzwingen? Also in Hana müssen es compounds sein, in psql müssen es singleColumnRanges sein

Reply:
Siehe oben, können wir gerne darüber sprechen weil mir das auch nicht gefällt wie es aktuell ist, aber sehe ich nicht im Rahmen dieses PRs, weil das mMn. sehr viele Änderungen wären.